packages/esm-patient-labs-app/src/lab-orders/add-lab-order/add-lab-order.test.tsx

```diff
@@ -1,14 +1,12 @@
 import React from 'react';
-import { fireEvent, render, renderHook, screen, waitFor, within } from '@testing-library/react';
+import { render, renderHook, screen, waitFor, within } from '@testing-library/react';
```

Comment:
I guess the `waitFor` import is now superfluous

Reply:
Not really, it is used when checking the orders
`
 await waitFor(() =>
      expect(hookResult.current.orders).toEqual([
        expect.objectContaining({
          urgency: 'STAT',
          instructions: 'plz do it thx',
          labReferenceNumber: 'lba-000124',
          testType: { label: 'CD4 COUNT', conceptUuid: 'test-lab-uuid-2' },
          careSetting: '6f0c9a92-6f24-11e3-af88-005056821db0',
          orderer: 'test-provider-uuid',
        }),
      ]),
    );
`

Reply:
Working fine for me without the `waitFor`:

![CleanShot 2024-02-09 at 1  00 32@2x](https://github.com/openmrs/openmrs-esm-patient-chart/assets/8509731/d436084d-373a-4ed2-a038-bdd42fbdc4e7)

Reply:
Welp. Looks like it doesn't work in the CI. Something to look into later.